README.md

```diff
@@ -122,6 +124,7 @@ Production capacities are centralized in the [capacities.json](https://github.co
 - Germany: [Fraunhoffer](https://energy-charts.de/power_inst.htm)
 - Great Britain: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Greece: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Guatemala: [AMM](http://www.amm.org.gt/pdfs2/2017/Capacidad_Instalada_2017.xls)
```

Comment:
I forgot to add the source when updating Guatemala capacities. I thought I could use this pull request to do it? Or should I do another PR?